vendor/k8s.io/kube-openapi/cmd/openapi-gen/openapi-gen.go

```diff
@@ -54,5 +54,4 @@ func main() {
 	); err != nil {
 		log.Fatalf("OpenAPI code generation error: %v", err)
 	}
-	log.Println("Code for OpenAPI definitions generated")
```

Comment:
oh, `k8s.io/kube-openapi` is not a staging repo - it's a real vendored repo. You need to fix this upstream.

Reply:
https://github.com/kubernetes/kube-openapi/pull/148